src/customisations/TchapComponentVisibility.ts

```diff
@@ -23,8 +24,12 @@ import { UIComponent } from "matrix-react-sdk/src/settings/UIFeature";
  * otherwise.
  */
 function shouldShowComponent(component: UIComponent): boolean {
+    const userId = MatrixClientPeg.get().getUserId();
+    const isExternal = userId.endsWith("agent.externe.tchap.gouv.fr") || userId.endsWith("agent.e.tchap.gouv.fr");
```

Comment:
It would be nice to have a function "isUserExternal" in the tchap files somewhere, so that we can reuse it later (we have not started extern customisation yet, but there will be many to do)

I would also add dev env, to avoid losing time with this when testing.

I don't know if there is a backend call we can do, or a property in MatrixClientPeg we can check, to know if a user is extern ? If we don't have anything clean like that, then this is fine.

Reply:
you mean to add a config var to identify which are the external server? I agree